gui/app/(tabs)/registration.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState } from 'react';
+import { View, Text, TextInput, StyleSheet, TouchableOpacity, Image } from 'react-native';
+import * as ImagePicker from 'expo-image-picker';
+
+interface RegistrationFormData {
+    firstName: string;
+    lastName: string;
+    username: string;
+    email: string;
+    password: string;
+    profilePicture: string | null;
+}
+
+const RegistrationScreen: React.FC = () => {
+    const [formData, setFormData] = useState<RegistrationFormData>({
+        firstName: '',
+        lastName: '',
+        username: '',
+        email: '',
+        password: '',
+        profilePicture: null,
+    });
+
+    const handleChoosePhoto = async () => {
+        const { status } = await ImagePicker.requestMediaLibraryPermissionsAsync();
+        if (status !== 'granted') {
+            alert('Sorry, we need camera roll permissions to make this work!');
+            return;
+        }
```

Comment:
Will this be triggered for desktop applications?

Reply:
No desktop app is not compatible. It is supported in Android Device, Android Emulator, iOS Device, iOS Simulator, Web. More details here https://docs.expo.dev/versions/latest/sdk/imagepicker/#platform-compatibility

Reply:
I said desktop, I should have said web. Thanks for clarifying. I suppose testing this would be just as good, but will this get triggered when accessed via web?

Reply:
yes